ospo-book/chapters/02-chapter.md

```diff
@@ -1 +1,107 @@
-TBD
\ No newline at end of file
+# Chapter 2: Understanding the Value of Open Source Program Offices
+
+- [Introduction](#introduction)
+    - Reasons Behind Starting an OSPO
+    - Reasons Behind Sustaining an OSPO
+- [Assessing Value of Open Source Activity](#assessing-value-of-open-source-activity) - `✅ Assessment`
+- [Antipatterns](#antipatterns) - `🚫 OSPO Antipatterns`
+- [Resources](#resources) - `📚 Continue Here`
+
+# Introduction
+
+From end-user companies and software companies to universities and public administrations, an organization's relationship with open source and the value it can get from an OSPO depend on the type of organization it is. However, we can still identify some critical problems that arise when getting involved in open source on an organizational level, which motivates the creation and continuity of an OSPO.
```

Comment:
Having to deal with open source is the reality for every organizations which uses software in some way. So I would see the argumentation not so much in challenges which arise when getting involved with open source, but necessary steps which have to be taken no matter what. To deal with software in a responsible and professional way requires to deal with open source. Doing this requires some organizational structure and that can an OSPO be.